File: Python/iniciante/1020idadedias.py
n = int(input())
ano = int(n / 365)
n -= ano * 365
mes = int(n / 30)
n -= int(mes * 30)
dia = n
print("%i ano(s)\n"
      "%i mes(es)\n"
      "%i dia(s)" % (ano, mes, dia))

File: C/Iniciante/1020idadedias.c
#include <stdio.h>
int main() {
    int n,ano,mes,dia;
    scanf("%i",&n);
    ano=n/365;
    n-=ano*365;
    mes=n/30;
    n-=mes*30;
    dia=n;
    printf("%i ano(s)\n"
           "%i mes(es)\n"
           "%i dia(s)\n",ano,mes,dia);
    return 0;
}